_docs_integrate/00-basics.md

```diff
@@ -33,4 +33,6 @@ If you like to try on your own: There is a Connector Tutorial in the next chapte
 
 # Support
 
-There is no general support for the Connector. Customer support can be addressed under `support[at]enmeshed.eu`.
+For assisted support with the Connector or the Backbone provided by the j&s-soft GmbH contact us via `support[at]enmeshed.eu`.
+
+Community support is a great way to get help and even contribute to the projects. Open bug reports and feature requests in the [Enmeshed Issue Tracker](https://github.com/nmshd/feedback/issues) or share your feedback with the Enmeshed team via the [Enmeshed Discussions](https://github.com/nmshd/feedback/discussions).
```

Comment:
sollte man das "contribute" nicht eher noch rauslassen, bis man wirklich contributen kann? Natürlich ist auch ein Feature Request in gewisser Weise eine Contribution, aber ich denke, man stellt sich i.d.R. was anderes drunter vor.

Reply:
Hm, ich habe das von unserem Issue Template übernommen weil wir das dort mal so gründlich ausformuliert haben. Ist ja zum aktuellen Zeitpunkt eher ne Floskel, da haste recht.